src/App.js

```diff
@@ -39,7 +39,7 @@ const withDefault = component => () => (
 
 class App extends Component {
   componentDidMount() {
-    history.replace('/');
+    // history.replace('/');
```

Comment:
Should we just delete this? I never understood why we would want to always redirect to the frontpage on the initial page load.

Reply:
I just read the other PR from @dawehner and now I understand why we need this.

Reply:
However, it's commented out here because it breaks routing right now.

Reply:
If you just run this from the dev server, and try and visit a page, you won't actually get anything to load.

Reply:
![broken-navigation](https://user-images.githubusercontent.com/1090713/38141249-10f7cf28-33ec-11e8-8291-7b6d24416442.gif)